install_config/aggregate_logging.adoc

```diff
@@ -1150,8 +1129,13 @@ For example:
 $ oc set env ds/logging-fluentd LOGGING_FILE_AGE=30 LOGGING_FILE_SIZE=1024000"
 ----
 
-Turn off log rotation by setting LOGGING_FILE_PATH=console. 
-This causes Fluentd to write logs to STDOUT where they can be retrieved using the `oc logs -f <pod_name>` command.
+Turn off log rotation by setting `LOGGING_FILE_PATH=console`. 
+This causes Fluentd to write logs to the Fluentd default location where you can retrieve them using the `oc logs -f <pod_name>` command.
+
+----
+oc set ds/fluentd LOGGING_FILE_PATH=console
```

Comment:
oc set env ds/logging-fluentd LOGGING_FILE_PATH=console

Reply:
@anpingli Thank you!!